webapp/components/Comment.vue

```diff
@@ -23,10 +23,27 @@
     <!-- eslint-disable vue/no-v-html -->
     <!-- TODO: replace editor content with tiptap render view -->
     <ds-space margin-bottom="small" />
-    <div style="padding-left: 40px;" v-html="comment.contentExcerpt" />
-    <!-- eslint-enable vue/no-v-html -->
+    <div v-if="!isActive" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
```

Comment:
The logic implemented in this template is calling for it's own component. How about `<collapsible>`?

Reply:
We have to refactor this anyway by using the editor library to display the content very soon. We should do this than together …